README.md

```diff
@@ -15,7 +15,7 @@ The template infra is intended to work with multiple application templates. See
 
 ## Installation
 
-To get started using the template infrastructure on your project, run the following command to execute the [install script](https://github.com/navapbc/template-infra/tree/main/template-only-bin/install-template.sh), which clones the template repository, copies the template files to your repository, and removes any files that are only relevant to the template itself:
+To get started using the template infrastructure on your project, run the following command in your project's directory to execute the [install script](https://github.com/navapbc/template-infra/tree/main/template-only-bin/install-template.sh), which clones the template repository, copies the template files to your repository, and removes any files that are only relevant to the template itself:
```

Comment:
Realized we should update this sentence reference the download-and-install-template.sh
```suggestion
To get started using the template infrastructure on your project, run the following command in your project's directory to execute the [download and install script](https://github.com/navapbc/template-infra/tree/main/template-only-bin/download-and-install-template.sh), which clones the template repository, copies the template files to your repository, and removes any files that are only relevant to the template itself:
```